Apidaze.SDK.Tests.Integration/IvrExample/Program.cs

```diff
@@ -0,0 +1,326 @@
+﻿using Apidaze.SDK.ScriptBuilder;
+using Apidaze.SDK.ScriptBuilder.POCO;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IvrExample
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                WebService.StartWebServer();
+
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message);
+            }
+            Console.ReadKey();
+        }
+    }
+
+    internal static class WebService
+    {
+        /// <summary>
+        /// The port the HttpListener should listen on
+        /// </summary>
+        private static readonly string SERVERL_URL = "http://01393488.ngrok.io";
+        private static readonly string LOCALHOST = "http://localhost:8080";
+
+        static readonly string INTRO = "/";
+        static readonly string STEP_1_PATH = "/step1.xml/";
+        static readonly string STEP_2_PATH = "/step2.xml/";
+        static readonly string STEP_3_PATH = "/step3.xml/";
+        static readonly string PLAYBACK_PATH = "/Apidazeintro.wav/";
```

Comment:
These are files, not directories. Therefore, it should be `/step1.xml` instead of `/step1.xml/`.